client/app/bundles/course/assessment/submission/submissions.js

```diff
@@ -1,25 +1,27 @@
-//= require helpers/url_helpers
-
-(function($, URL_HELPERS) {
-  'use strict';
-  var DOCUMENT_SELECTOR = '.course-assessment-submission-submissions.edit ';
-  var SINGLE_QUESTION_ASSESSMENT_SELECTOR = DOCUMENT_SELECTOR + '.single-question ';
-  var MULTI_QUESTION_ASSESSMENT_SELECTOR = DOCUMENT_SELECTOR + '.multi-question ';
-  var SUMMARY_GRADE_SELECTOR = '.submission-grades-summary-grade';
-  var GRADE_INPUT_SELECTOR = 'input.grade';
-  var POINTS_AWARDED_SELECTOR = 'input.submission-points-awarded';
-  var MAXIMUM_GRADE_SELECTOR = '.submission-statistics-maximum-grade';
-  var TOTAL_GRADE_SELECTOR = '.submission-statistics-total-grade';
+import getUrlParameter from 'lib/helpers/url_helpers';
+
+/* eslint wrap-iife: ["error", "inside"] */
+/* eslint no-use-before-define: ["error", { "functions": false }] */
+/* global $:false, jQuery:false */
+(function ($) {
+  const DOCUMENT_SELECTOR = '.course-assessment-submission-submissions.edit ';
```

Comment:
`DOCUMENT_SELECTOR`  is the way we were doing page specific javascript before, now it can be removed as well.